api-dummy/public/schemas/libero/core/inline.rng

```diff
@@ -27,6 +27,13 @@
         </element>
     </define>
 
+    <define name="libero.text.foreign">
+        <element name="foreign">
```

Comment:
not sure `foreign` is the right term here, it could be a second or bilingual language for the reader? Works as a placeholder for now

Reply:
Foreign as in different to the surrounding text ([borrowed from TEI](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-foreign.html)). But yeah, these are all placeholders anyhow.